apps/site/assets/css/_header.scss

```diff
@@ -310,10 +394,16 @@ header.header.menu-open {
   }
   .m-menu__content {
     visibility: unset;
-    width: 220px;
+    width: auto;
     @include media-breakpoint-only(xs) {
       width: 100%;
     }
+    @include media-breakpoint-only(sm) {
+      width: 360px;
+    }
+    @include media-breakpoint-only(md) {
+      width: 384px;
```

Comment:
Would a flexible value for the width make sense here because it's getting larger for larger breakpoints anyway? 🤔

Reply:
What should the scale factor be?  I feel like the style would be harder to control if it scaled as a percentage of the viewport

Reply:
I was thinking more a percentage of its _container_ rather than the viewport, or maybe a `calc(100% - {however big the space on the right of the section is})`. But let's defer it to the styling ticket anyway.